awx/main/models/base.py

```diff
@@ -61,7 +60,10 @@
     (PERM_INVENTORY_CHECK, _('Check')),
 ]
 
-CLOUD_INVENTORY_SOURCES = list(CLOUD_PROVIDERS) + ['scm']
+
+def discover_cloud_inventory_sources():
+    return list(discover_available_cloud_provider_plugin_names()) + ['scm']
```

Comment:
It already returns a list, according to the type annotations, so it's enough to do
```suggestion
    return discover_available_cloud_provider_plugin_names() + ['scm']
```
or
```suggestion
    return [*discover_available_cloud_provider_plugin_names(), 'scm']
```